clang/test/CodeGenSYCL/intel-fpga-loops.cpp

```diff
@@ -171,6 +174,27 @@ void max_reinvocation_delay() {
       a[i] = 0;
 }
 
+// Add codeGen tests for loop attribute: [[intel::enable_loop_pipelining]].
+void fpga_enable_loop_pipelining() {
+  int a[10];
+  // CHECK: ![[MD_FP]] = distinct !{![[MD_FP]], ![[MP]], ![[MD_fpga_pipeline:[0-9]+]]}
+  // CHECK-NEXT: ![[MD_fpga_pipeline]] = !{!"llvm.loop.intel.pipelining.enable", i32 1}
+  [[intel::enable_loop_pipelining]] for (int i = 0; i != 10; ++i)
+    a[i] = 0;
+
+  // CHECK: ![[MD_FP_1]] = distinct !{![[MD_FP_1]], ![[MP]], ![[MD_fpga_pipeline]]}
```

Comment:
Shouldn't there be a check for 'llvm.loop.intel.pipelining.enable' below?

Reply:
Thanks @premanandrao for feedbacks.

IR looks like:
```
!72 = distinct !{!72, !14, !73}
!73 = !{!"llvm.loop.intel.pipelining.enable", i32 1}
!74 = distinct !{!74, !14, !73}
!75 = distinct !{!75, !14, !73}
```
The attribute generates same metadata when it applies to for and other loops. So i added CHECK for 'llvm.loop.intel.pipelining.enable'  at line 181.

Reply:
Oh, makes sense.